app/index.js

```diff
@@ -25,12 +25,15 @@ nunjucks.configure(path.join(__dirname, 'views'), {
 app.use(require('cookie-parser')());
 app.use(require('express-session')(config.session));
 
-passport.use(new Auth0Strategy(
+strategy = new Auth0Strategy(
```

Comment:
This sould be `const strategy = ` or at least `var strategy = `

Reply:
Good point 👍